test/test_run_detext.py

```diff
@@ -0,0 +1,75 @@
+"""Tests for run_detext."""
+import os
+import shutil
+import tensorflow as tf
+import sys
+
+from detext.run_detext import main
+
+
+class TestModel(tf.test.TestCase):
```

Comment:
What is the difference between this test and the run_detext*.sh files?

Reply:
It's increasing the test coverage which is important as I found several issues that not covered with the existing tests. For example, the filter window size format in #28, the error of not allowing unknown args (which was also discovered during the integration tests).

Reply:
Does it mean that run_detext*.sh can be a subset of this test?

If so, we can move tests covered in run_detext*.sh to this test since this test can be automated by running the `pytest` command

Reply:
Sure, I can convert them to the tests.

Reply:
Agreed. Having an integration test is necessary 👍 
(Please keep a sh run script for the toy example.)

Reply:
Done